src/main/java/kg/jarkyn/cobspecserver/Runner.java

```diff
@@ -0,0 +1,32 @@
+package kg.jarkyn.cobspecserver;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+public class Runner {
+    public static void main(String[] args) {
+        server().run();
+    }
+
+    private static Server server() {
+        return new Server(listener(), controller());
+    }
+
+    private static ListenerSocket listener() {
+        return new ListenerSocket(getServerSocket(), true);
```

Comment:
Never had the flag to begin with, was initialising this value to true internally because I would never have initialised with false, but then received feedback to pass into constructor.

this was initial Runner implementation:
https://github.com/indykid/cobSpecServer/commit/3a1eb6b7eb5d6d3661524a19e523b9d8e29fcc41

this was initial ListenerSocket implementation:
https://github.com/indykid/cobSpecServer/blob/be5778a77bab92ecde094e4a8ddcc13f43dbbe0e/src/main/java/kg/jarkyn/cobspecserver/ListenerSocket.java

I personally prefer my initial implementation, felt super weird to pass this boolean flag in, especially after reading Clean Coder, where he mentions that passing flags as args is a smell. But the feedback came from two people, so I didn't stand my ground :)